source/exceptions.tex

```diff
@@ -365,12 +365,12 @@
 \indextext{unwinding!stack}%
 As control passes from the point where an exception is thrown
 to a handler,
-destructors are invoked by a process, specified in this subclause, called
-\defn{stack unwinding}.
+objects with automatic storage duration are destroyed by a process,
+specified in this subclause, called \defn{stack unwinding}.
 
 \pnum
-The destructor is invoked for each automatic object of class type constructed,
-but not yet destroyed,
+Each object with automatic storage duration is destroyed if it has been
```

Comment:
Does not look like semantic line break (https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#source-code-formatting)